internal/pkg/deploy/cloudformation/app.go

```diff
@@ -14,20 +14,28 @@ import (
 
 // DeployApp wraps the application deployment flow and handles orchestration of
 // creating a stack versus updating a stack.
-func (cf CloudFormation) DeployApp(template, stackName, changeSetName, cfExecutionRole string, tags map[string]string) error {
+func (cf CloudFormation) DeployApp(template, stackName, changeSetName, cfExecutionRole string, tags map[string]string, parameters map[string]string) error {
```

Comment:
After this change, let's not make add any more functionality to this package without refactoring it first.

I'm not a fan of this signature because it's:
1. different than the rest of the methods in this pkg.
2. it's not possible to specify any of these parameters as optional.

Reply:
:+1: I hate myself when I wrote add another parameter to the long-enough parameter list

Reply:
🤣🤣🤣🤣🤣🤣🤣